### PR TITLE
Update readme with build options

### PR DESCRIPTION
**Pull request by Sherry-Lin**
Add instroduction for ENABLE_KERNELS/FREE_KERNELS/BUILD_KERNELS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,18 +150,62 @@ Supported input format and max resolution:
...
-|                   | Output |  Y   |  Y   |  Y   |  Y   |  Y   |      |  Y   |  Y   |  Y   |
+| Chroma Siting                                |     |  Y  |    Y    |  Y  |  Y  |  Y  |  Y  |
+
+For detail feature information, you can access [Media Features](https://github.com/intel/media-driver/blob/master/docs/media_features.md).
+
+## HW Media Features
+
+| Media Features      | BDW  | SKL  | BXT/APL |   KBL   |   CFL   | CNL  |   ICL*  |
+|------------|------|------|---------|---------|---------|------|---------|
+| AVC        | D | D | D/E  | D/E  | D/E  | D | D/E  |
```
Comment (dvrogozh) [inline README.md]: We should have VDENC encoding on all Gen9, including SKL (#465). Please, fix.

File: docs/media_features.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Media Features Information
```
Comment (dvrogozh) [inline docs/media_features.md]: Better: "Media Features Summary".

**Comment by dvrogozh** [comment]
@Sherry-Lin : can you, please, provide updated PR?

File: README.md [hunk 3]
```diff
@@ -216,18 +150,62 @@ Supported input format and max resolution:
...
+| MPEG-2     | D | D | D       | D    | D    | D | D    |
+| VC-1       | D    | D    | D       | D       | D       | D    | D       |
+| JPEG       | D    | D/E  | D/E     | D/E     | D/E     | D/E  | D/E     |
+| VP8        | D    | D    | D       | D       | D       | D | D    |
+| HEVC 8bit  |      | D | D    | D    | D    | D | D/E  |
+| HEVC 10bit |      |      | D       | D       | D       | D | D/E  |
+| VP9 8bit   |      |      | D       | D       | D       | D    | D/E     |
+| VP9 10bit  |      |      |         | D       | D       | D    | D/E     |
+
```
[the remark this answers is not in the document]
Comment (Sherry-Lin) [reply README.md]: VP features are controlled by driver code which depends on input parameter and execution platform, so we only support these HW features when ENABLE_KERNELS=OFF.

**Pull request merged**